guide/designing-products/personal-finance.md

```diff
@@ -132,7 +132,7 @@ The amounts involved in these transactions are higher and therefore require bett
    height = 800
 %}
 
-A solution for Bitcoin could be a hardware wallet with an accompanying desktop application. While keys are stored, and transactions get signed on secure hardware, users can still enjoy a desktop application's convenience. This avoids or reduces some of the risks involved with the mobile wallet configuration described in the first category while adding a small amount of friction acceptable for transactions that occur less frequently.
+A hardware wallet accompanied by a desktop application where keys are stored and transactions are securely signed by hardware is another solution. This reduces the risk of keeping funds on a mobile wallet configuration but adds acceptable friction for transactions that occur less frequently.
```

Comment:
Does not make sense anymore after the changes. The previous paragraph talks about how this typically works when using a bank for these transactions. This paragraph talks about how you would do this with Bitcoin. So it's not `another solution`.

Reply:
Good catch, so one way in which I tried to fix it is by removing that reference in https://github.com/BitcoinDesign/Guide/pull/374/commits/a9be74dce4bc3cbf5be164e4504eb70b01ca63d5 another way would be to revert both of those parts, let me know which one you prefer.